odoo/addons/base/models/res_users.py

```diff
@@ -1518,7 +1518,11 @@ class CheckIdentity(models.TransientModel):
 
     def run_check(self):
         assert request, "This method can only be accessed over HTTP"
-        self.create_uid._check_credentials(self.password, {'interactive': True})
+        try:
+            self.create_uid._check_credentials(self.password, {'interactive': True})
+        except AccessDenied:
+            raise UserError(_("Incorrect Password, try again or click on Forgot Password to reset your password."))
```

Comment:
there is no "Forgot Password" button when you are configuring your authentication method in the backend